src/harness/testcases/registration_testcase.py

```diff
@@ -67,6 +69,7 @@ def test_10_3_4_2_5_1(self):
     # Register the device, make sure at least one required parameter is invalid
     device_a = json.load(
         open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
```

Comment:
I think you want to switch line 72 and line 73. I realized it after I approved the pull request. Can submit the fix with next pull request.